projects/chaos/src/power_path.c

```diff
@@ -20,10 +29,21 @@ static bool prv_addr_eq(GPIOAddress addr1, GPIOAddress addr2) {
 // Interrupt handler for over and under voltage warnings.
 static void prv_voltage_warning(const GPIOAddress *addr, void *context) {
   PowerPathCfg *pp = context;
+  PowerPathVCReadings readings = { 0 };
+  StatusCode status = STATUS_CODE_OK;
   if (prv_addr_eq(*addr, pp->dcdc.uv_ov_pin) && pp->dcdc.monitoring_active) {
-    event_raise(CHAOS_EVENT_CAN_UV_OV, POWER_PATH_SOURCE_ID_DCDC);
+    power_path_read_source(&pp->dcdc, &readings);
+    LOG_DEBUG("DCDC\n");
```

Comment:
If we want to keep these around, we should probably look into removing these at compile time. Otherwise, if these are just for testing, then these can probably be removed?

Reply:
Done. Removed.